docs/conf.py

```diff
@@ -117,8 +117,8 @@ def setup(app):
 
     # Headers for columns to translate in codelist CSVs
     codelist_headers = ['Title', 'Description', 'Extension']
-    # Headers for columns to translate in mapping CSVs
-    mapping_headers = ['Mapping to OC for Infrastructure', 'Mapping from OCDS']
+    # Headers for columns to translate in mapping CSVs. The headers in babel_ocds_mapping.cfg  should match these
```

Comment:
Can also say the same about `codelist_headers` above and `babel_ocds_codelist.cfg`.

```suggestion
    # Headers for columns to translate in mapping CSVs. The headers in babel_ocds_mapping.cfg should match these.
```

Reply:
Done in https://github.com/open-contracting/infrastructure/pull/340/commits/6e629fbe3c197b476a98053b7087f6b0ac9e6d43